scripts/deploy/govcms-enable-modules

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS enable required modules
+#
+# The GovCMS platform provides some modules to assist with running on
+# the platform, this will ensure they're enabled every deploy.
+#
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_DEPLOY_ENABLE_MODULES=${GOVCMS_DEPLOY_ENABLE_MODULES:-true}
+
+if [ "$GOVCMS_DEPLOY_ENABLE_MODULES" != "true" ]; then
+  echo "[Skip]: Enable default modules for GovCMS"
+  exit 0
+fi
+
+echo "GovCMS Deploy :: Enable modules"
+
+MODULES=$(drush pm:list --status=enabled)
+NON_PROD_MODULES=("stage_file_proxy")
+PLATFORM_MODULES=(
+  "redis"
+  "fast404"
+  "clamav"
+  "robotstxt"
+  "lagoon_logs"
+  "environment_indicator"
+)
+
+if [ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]; then
+  for MODULE in "${NON_PROD_MODULES[@]}"; do
+    if [[ $(echo "$MODULES" | grep -c "$MODULE") -eq 0 ]]; then
+      drush pm:enable "$MODULE" -y
+    fi
+  done
+fi
+
+# Enable Lagoon required modules in production.
+for MODULE in "${PLATFORM_MODULES[@]}"; do
+    if [[ $(echo "$MODULES" | grep -c "$MODULE") -eq 0 ]]; then
+    drush pm:enable "$MODULE" -y
```

Comment:
is there a time difference between enabling them in a loop, or as a single string list?

Reply:
I was actually thinking that - I'll try and time it and see how we go.

Reply:
```
Successfully enabled: stage_file_proxy, redis, fast404, clamav, robotstxt, lagoon_logs, environment_indicator
real	0m7.263s
user	0m4.259s
sys	0m1.022s
```

```
 [success] Successfully enabled: stage_file_proxy
 [success] Successfully enabled: redis
 [success] Successfully enabled: fast404
 [success] Successfully enabled: clamav
 [success] Successfully enabled: robotstxt
 [success] Successfully enabled: lagoon_logs
 [success] Successfully enabled: environment_indicator

real	0m19.370s
user	0m11.471s
sys	0m3.117s
```

Reply:
Enabling in one go is a winner - updated.